### DISCUSSION TITLE
Repseqio status

### DISCUSSION THREAD

**Discussion by Thopic** [milaboratory/mixcr#1626] [Q&A] [answered]
Hi everyone,

What is the current status of repseqio ? I want to use "repseqio compile" to get the sequences from the json file (after having run findAlleles) but I don't know if the software is still available (I found an old version on zenodo, but the github link seems to be dead).

Is there a version avaible somewhere or an alternative to extract sequences from repseqio json files  ?

Thanks a lot,
Thomas

**Comment by mizraelson** [comment, marked as answer]
Hi, you can actually export the library directly into a FASTA file from the `findAlleles` command using the parameter:

`--export-library alleles_library.fasta`

**Reply by Thopic** [reply]
Oh, thanks a lot! It's perfect
